packages/fx-core/tests/plugins/solution/solution.migrate.test.ts

```diff
@@ -146,9 +146,6 @@ describe("Solution migrate()", async () => {
     expect(mockedSolutionCtx.envInfo.state.get(GLOBAL_CONFIG)).is.not.undefined;
     const lang = mockedSolutionCtx.projectSettings?.programmingLanguage;
     expect(lang).equals("javascript");
-    expect(fileContent.has("./.vscode/launch.json")).to.be.true;
```

Comment:
If the test case is still needed, change this to `to.be.false`, otherwise just removing the whole test case.

Reply:
I try to remove the shole test case but failed because of the test coverage. It will be removed in the  core & solution PR too.